### ISSUE TITLE
Change the inserter search results message to `polite`

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#13387]
When opening the inserter, the number of initial available results is announced by assistive technologies via a `speak()` message sent to the _assertive_ ARIA live region.

The assertive message interrupts what's currently being announced by the screen reader and, with _some_ browser / screen reader combinations, this prevents the search field to be announced correctly.

In my testing, Safari + VoiceOver are affected. To reproduce:
- using the keyboard, tab to the main inserter button
- press Control Option Spacebar
- the inserter opens, focus is moved to the search field
- the search field label "Search for a block" is not announced
- the number of results is announced almost immediately

![screenshot 2019-01-20 at 17 01 53](https://user-images.githubusercontent.com/1682452/51442901-11e64100-1ce2-11e9-9f3c-014b54671a5b.png)

Changing the message from `assertive` to `polite` fixes the issue and in my testing doesn't affect other browser / screen reader combos.

Worth noting this message is debounced by 500 ms and this might explain the different behavior across different combos. Also, screen readers have different ARIA live implementations. Regardless, it makes sense to let screen readers announce the search field first, and then the number of results.
